.buildkite/pipeline.yml

```
@@ -245,6 +245,7 @@
           - AWS_ACCESS_KEY_ID
           - AWS_SECRET_ACCESS_KEY
           - AWS_SESSION_TOKEN
+          - IS_CI=true
```

We already have an env var for this which seems to be standard across CI providers
```suggestion
          - CI
```